### ISSUE TITLE
Free pricing not validated properly

### ISSUE THREAD

**Issue by mihaisc** [oceanprotocol/market#1060]
![image](https://user-images.githubusercontent.com/26000280/152553021-cb37051b-a4a5-4c2e-9d50-f3613b8b92c9.png)
If i change the free price for example then switch to free again and move to the next step i get 
![image](https://user-images.githubusercontent.com/26000280/152553166-b4daf391-4b29-49a9-a910-a79a7c5148ae.png)

**Comment by kremalicious** [comment]
more issues with free pricing screen:

- "Continue" button stays disabled, you're not able to continue in the "FREE" tier
- either we add some user intention like a checkbox ("I want this asset to be free", or "I understand network fees are still to be paid"), or automatically make pricing valid when user hits the free screen

**Comment by EnzoVezzaro** [comment]
if it's not a problem, I can take this.

**Comment by EnzoVezzaro** [comment]
issue on the layout is done, now working on the issue 2 (pricing don't show up)

**Comment by mihaisc** [comment]
No, that is not an issue here , is already handed in the restore order pr.
